`add` command fails with NOT NULL error for valid input

# Description
Running a valid `add` command fails with a database error.

```bash
lynx add alias uri
```

## Actual result

```bash
[ERROR] NOT NULL constraint failed: bookmarks.uri
```

## Expected result
The bookmark should be added successfully.

--- a/lynx.c
+++ b/lynx.c
@@ -47,7 +47,7 @@ bool add_bookmark(sqlite3 *db, char *alias, char *uri) {
   if(sqlite3_bind_text(stmt, 1, alias, -1, SQLITE_STATIC) != SQLITE_OK) {
     nob_log(ERROR, "could not bind %s because of %s", alias, sqlite3_errmsg(db));
   }
-  if(sqlite3_bind_text(stmt, 1, uri, -1, SQLITE_STATIC) != SQLITE_OK) {
+  if(sqlite3_bind_text(stmt, 2, uri, -1, SQLITE_STATIC) != SQLITE_OK) {
     nob_log(ERROR, "could not bind %s because of %s", uri, sqlite3_errmsg(db));
   }
 

